src/app/components/GoalScorerDisplay.tsx

```diff
@@ -2,14 +2,11 @@ import React, { SyntheticEvent, Dispatch, useState } from "react";
 import { TeamScore } from "./TeamScorer";
 
 type Props = {
-  teamAScoreObject: TeamScore;
```

Comment:
Since React is based around components, you can just have a single GoalScorerDisplay component which you can use twice for both teams. They just have different props to represent the data of the two different teams, but the render logic is identical